utils/obsproc/Ghrsst2Ioda.h

```diff
@@ -157,21 +157,33 @@ namespace gdasapp {
       std::vector<std::vector<float>> lon2d_s;
       std::vector<std::vector<float>> lat2d_s;
       std::vector<std::vector<float>> obserror_s;
-      std::vector<std::vector<float>> seconds_s;
       if ( fullConfig_.has("binning") ) {
         sst_s = gdasapp::superobutils::subsample2D(sst, mask, fullConfig_);
         lon2d_s = gdasapp::superobutils::subsample2D(lon2d, mask, fullConfig_);
         lat2d_s = gdasapp::superobutils::subsample2D(lat2d, mask, fullConfig_);
         obserror_s = gdasapp::superobutils::subsample2D(obserror, mask, fullConfig_);
-        seconds_s = gdasapp::superobutils::subsample2D(seconds, mask, fullConfig_);
       } else {
         sst_s = sst;
         lon2d_s = lon2d;
         lat2d_s = lat2d;
         obserror_s = obserror;
-        seconds_s = seconds;
       }
 
+      // Non-Superobing part only applied to datetime
+      // TODO(Mindo): Think superob datetime efficiently
+      int64_t sum = 0;
```

Comment:
Can't you use ```std::accumulate``` to compute the mean if you can't use ```.sum()```? I think it's in ```numeric``` (#include <numeric>).